source/_components/opentherm_gw.markdown

```diff
@@ -364,8 +163,319 @@ The value you provide here will be used with the GPIO `home` (5) and `away` (6)
 
 | Service data attribute | Optional | Description |
 | ---------------------- | -------- | ----------- |
+| `gateway_id` | no | The `gateway_id` as specified in `configuration.yaml`.
 | `temperature` | no  | The setback temperature. Accepted values are `0.0` through `30.0`.
 
+## {% linkable_title Sensors %}
+
+The following `sensor` entities will be created for each configured gateway. The `entity_id` of every sensor will have a suffix containing the `gateway_id` of the gateway to which it belongs.
+<p class='note'>
+Not all boilers and thermostats properly support all OpenTherm features, so not all of the sensors will have useful values.
+</p>
+
+- <b>burner_hours</b><br />
```

Comment:
`<br>`
br has no end tag I think.

Maybe replace those with simple line breaks in the markdown?

Reply:
The bold format will be changed to the stars notation.

`<br>` has no end tag, the slash is there to indicate a void element. Either `<br>` `<br/>` or `<br />` would do the same, with the latter being the most compatible as far as I know (not sure if that's relevant in this case though).
A line break alone would not translate to the additional info being on a new line. The markdown solution for this would be to replace the `<br />` tags with 2 trailing spaces, but personally I am not a big fan of trailing whitespaces as part of any syntax.
If, however, this is the preferred way to go within Home Assistant, I will change it.

Reply:
Sounds good!

Reply:
Changed it to the trailing whitespaces in the last commit, it's a markdown file after all...